src/services/order.ts

```diff
@@ -0,0 +1,14 @@
+import {Order} from "../models/Order";
+
+export const createOrder = async (order:OrderAttributes)=>{
+  return await Order.create({ ...order });
+};
+
+
+interface OrderAttributes {
+    email: string,
+    status: ORDER_STATUS,
```

Comment:
Where `ORDER_STATUS` has been imported in this page?

Reply:
I added imports statement 👍